pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/BrokerRegistryImpl.java

```diff
@@ -122,7 +120,7 @@ public boolean isStarted() {
     public synchronized void register() throws MetadataStoreException {
         if (this.state == State.Started) {
             try {
-                this.brokerLookupDataLock = brokerLookupDataLockManager.acquireLock(keyPath(brokerId), brokerLookupData)
+                brokerLookupDataMetadataCache.put(brokerIdKeyPath, brokerLookupData, EnumSet.of(CreateOption.Ephemeral))
```

Comment:
In https://github.com/apache/pulsar/issues/23889#issuecomment-2614188629 @Joforde shared an observation that the `CreateOption.Ephemeral` will be ignored and a persistent node will get created. @BewareMyPower @heesung-sn Please check this.

Reply:
Will be addressed by PR #23902